onnxruntime/core/session/inference_session.cc

```diff
@@ -365,6 +546,25 @@ common::Status InferenceSession::Load(const void* model_data, int model_data_len
   return Load(loader, "model_loading_array");
 }
 
+common::Status InferenceSession::Load() {
+  if (model_proto_ == nullptr) {
+    return ORT_MAKE_STATUS(ONNXRUNTIME, FAIL, "ModelProto corresponding to the model to be loaded has not been parsed yet.");
+  }
```

Comment:
'has not been parsed yet' is a little vague. Should this say that a different InferenceSession ctor should have been called?

Reply:
Sure, modified the message a bit.